src/config_parser.cpp

```diff
@@ -257,6 +255,10 @@ namespace
 
     void LoadMqttToSmartWebConfig(TConfig& config, const Json::Value& configJson)
     {
+        if(configJson.isMember("debug")) {
```

Comment:
```suggestion
        if (configJson.isMember("debug")) {
```
`clang-format` будет ругаться

Reply:
Поправил